stable/README.md

```diff
@@ -1,167 +1,167 @@
+| transparent-hugepage| seLinuxContext.* | To mount hostPath and disable THP on host|
+| admin, database| allowedCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
+| admin, database| defaultAddCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
 
-1. Download your [desired version][2]
-2. Unpack it (`tar -zxvf helm-${helm-version}-linux-amd64.tgz`)
-3. Find the helm binary in the unpacked directory, and move it to its desired destination (`mv linux-amd64/helm /usr/local/bin/helm`)
 
-From there, you should be able to run the client: `helm help`.
+## Install Helm and Tiller
 
-We’ll use Helm version 2.14.1, which can be downloaded via <https://github.com/kubernetes/helm/releases/tag/v2.14.1.>
+There are two parts to Helm: The Helm client (`helm`) and the Helm server (`tiller`). 
 
-Run the following commands to install the Helm client:
+Every [release][2] of Helm provides binary releases for a variety of OSes. 
 
+1. Download your [desired version][2]
+2. Unpack it (`tar -zxvf helm-${helm-version}-linux-amd64.tgz`)
+
+We’ll use Helm version 2.16.1, which can be downloaded via <https://github.com/kubernetes/helm/releases/tag/v2.16.1.>
+
+Run the following commands to install the Helm locally on your Linux client machine:
 ```bash
-$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-linux-amd64.tar.gz | tar xz
+$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-linux-amd64.tar.gz | tar xz
 $ cd linux-amd64
-$ mv helm /usr/local/bin
-$ mv tiller /usr/local/bin
 ```
 
-If you're running on Mac, the curl commands above would be:
-
+If you're running on Mac, the commands are:
 ```bash
-$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-darwin-amd64.tar.gz | tar xz
+$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-darwin-amd64.tar.gz | tar xz
 $ cd darwin-amd64
+```
+
+Move the Helm binaries to /usr/local/bin
+```
 $ mv helm /usr/local/bin
 $ mv tiller /usr/local/bin
 ```
 
-The following command will configure the current users environment for Helm; it will create a `.helm` directory in `${HOME}`.
+Confirm you can run the Helm client: `helm help`.
 
-```bash
-$ helm init --client-only
-$HELM_HOME has been configured at /home/clusteradmin/.helm.
-Not installing Tiller due to 'client-only' flag having been set
-Happy Helming!
-```
 
-Alternatively, to keep isolated between individuals sharing a login, you can have separate Kube, Helm, and Tiller state (so run these before the above init command):
+## Create the _nuodb_ namespace to install NuoDB
 
-```bash
-export TILLER_NAMESPACE=nuodb
-export HELM_HOME=`pwd`/.helm
-export KUBECONFIG=`pwd`/.kube/config
+```
+kubectl create namespace nuodb
 ```
 
-### Install the Tiller Server
-
-We will be creating the Tiller server in the `kube-system` namespace so that it is available to all projects.
+## Create the _tiller_ service account in the _kube-system_ namespace 
```

Comment:
only required in OCP. Not clear from this wording

Reply:
when using Helm init for K8s I get the same results -- Tiller gets installed in the kube-system namespace. If something needs changing, suggest approve the changes and make adjustments in the ENG doc PR.